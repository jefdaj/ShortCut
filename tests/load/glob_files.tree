.
|-- cache
|-- exprs
|   |-- glob_files
|   |   `-- e98b828d23.str.list -> ../../cache/lines/...
|   `-- str
|       `-- 4749080868.str -> ../../cache/lines/...
`-- vars
    `-- result -> ../exprs/glob_files/e98b828d23.str.list

5 directories, 3 files
