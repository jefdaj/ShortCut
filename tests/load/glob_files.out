["data/6803genes.faa",
 "data/6803genes10.faa",
 "data/7002genes.faa",
 "data/7122genes.faa",
 "data/7942gene.faa",
 "data/7942genes.faa",
 "data/7942genes10.faa",
 "data/7942genes50.faa",
 "data/Mycoplasma_agalactiae_small.faa",
 "data/Mycoplasma_genitalium_small.faa",
 "data/Mycoplasma_hyopneumoniae_small.faa"]
